File: app/src/main/java/com/ekm/hairdo/things/favs.java
package com.ekm.hairdo.things;

import java.util.ArrayList;

public class favs {


    public ArrayList<String> mFavs;

    public favs() {
        this.mFavs = new ArrayList<>();
    }

    public ArrayList<String> getmFavs() {
        return mFavs;
    }

    public void setmFavs(ArrayList<String> mFavs) {
        this.mFavs = mFavs;
    }

    public void addToFav(String id) {
        mFavs.add(id);
    }

    public  void  removeFav(String id) {
        mFavs.remove(id);
    }
}

File: app/src/main/java/com/ekm/hairdo/things/favs.kt
package com.ekm.hairdo.things

import java.util.ArrayList

class favs {

    var mFavs: ArrayList<String>? = null

    fun favs() {
        mFavs = ArrayList()
    }

    fun addToFav(id: String) {
        mFavs!!.add(id)
    }

    fun removeFav(id: String?) {
        mFavs!!.remove(id)
    }
}